docs/usage/configuration-options.md

```diff
@@ -2497,6 +2476,27 @@ This will lead to following update where `1.21-alpine` is the newest version of
 image: my.new.registry/aRepository/andImage:1.21-alpine
 ```
 
+## registryAliases
```

Comment:
How can I preview this on the `renovatebot.io` repository? I think there was something about using `git submodule` to pull in the fork branch?

It's hard for me to see what's changing from within the GitHub interface. 😄

Reply:
I haven't worked on `renovatebot.github.io`, but from what I see you should be able to run it locally with:  

```shell
#inside of the clone repository 
git submodule update --init --recursive

make

mkdocs serve
```